src/effects/visibleeffectslist.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <QList>
+
+#include "effects/backends/effectmanifest.h"
+
+class QDomDocument;
```

Comment:
```suggestion
QT_FORWARD_DECLARE_CLASS(QDomDocument);
```

Reply:
I cannot find documentation for this macro. Why use it rather than a standard forward declaration?

Reply:
https://github.com/mixxxdj/mixxx/pull/3052
https://beefdev.blogspot.com/2011/12/qtforwarddeclareclass-should-be-used.html

Reply:
This macro seems to be useless. From qglobal.h:
```
# define QT_FORWARD_DECLARE_CLASS(name) class name;
```